README.md

```diff
@@ -20,9 +20,7 @@ bundle install
 touch .env
 
 # Clone production data
-rake db:create db:migrate
-rake cascade:sync
-rake faculty:sync
+cap production db:clone
```

Comment:
@tatwell to also ease your worries, we have this functionality as a Capistrano task on Inside as well. Not once have we ever had an issue with any task being run at any point when it wasn't intended to be.